pandas/tests/frame/indexing/test_getitem.py

```diff
@@ -67,6 +67,14 @@ def test_getitem_sparse_column_return_type_and_dtype(self):
         result = df.loc[:, "A"]
         tm.assert_series_equal(result, expected)
 
+    def test_getitem_indexer_empty_series(self):
+        # GH: 19290
+        df = DataFrame(columns=["a", "b", "c"])
+        ind = df.a.apply(lambda x: x == "0")
```

Comment:
is there any way to do this without `apply`?  we want it to be really obvious what is being tested

Reply:
Yes, wanted to keep the issue structure. Changed it